### DISCUSSION TITLE
Adam Wathan - Tailwind

### DISCUSSION THREAD

**Discussion by hipstersmoothie** [devtools-fm/devtools.fm#278] [Announcements]
- https://adamwathan.me/
- https://github.com/adamwathan
- https://twitter.com/adamwathan

### Tailwind

- [x] What is tailwind? What are the benefits of the approach?
- [x] How did it first start? How has that definition evolved over the years?
- [x] What's new in v4?
- [x] What portions did you move to rust?
- [x] What are css layers and how does tailwind use them?
- [x] What are custom properties and how does tailwind use them?
- [x] You've been full time on tailwind for half a decade. What are some of the ways you've tried to sustain it and what has worked?
- [x] Tailwind is oddly a pretty divisive technology. Why do you think people have such a reaction to it? Do you think any of their points ring true?
- [ ] ~What is headless UI? Do you like any of the other headless UI react libraries?~
- [ ] ~What are your favorite modern CSS features?~

### Future

- [ ] What do you have planned next for tailwind?
- [ ] ~What do you think the future of styling is?~

**Comment by just-be-dev** [comment]
Tooltips:
- https://grats.capt.dev/
- https://github.com/teableio/teable

**Comment by hipstersmoothie** [comment]
https://www.onedoclabs.com/
https://github.com/daytonaio/daytona
